### PR TITLE
[github actions] add minimal workflow to test token

### PR DESCRIPTION
**Pull request by XilaiZhang**
Add a workflow that simply:
1. checkout repo
2. write a file
3. create a PR

This is used to verify the permissions and validity of the github token we use to create PR.
context: https://chat.google.com/room/AAAAc_4rqiI/Ck593Sg7mvs

If we think the triggering condition is complicated, I can also change this workflow to be manual dispatch, to make it truly minimal.

### CODE REVIEW

File: .github/workflows/easy-cp.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
           base: ${{ env.RELEASE_BRANCH }}
           branch: cp-${{ env.CHANNEL }}-${{ env.COMMIT_SHA }}
           path: flutter
-          token: ${{ secrets.FLUTTERMIRRORINGBOT_TOKEN}}
```
Comment (godofredoc) [inline .github/workflows/easy-cp.yml:63]: Let's use `secrets.FLUTTERACTIONSBOT_CP_TOKEN`

File: .github/workflows/minimal.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        run: |
+          cd flutter
+          touch empty.json
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38
+        with:
+          path: flutter
+          commit-message: blah
+          committer: GitHub <noreply@github.com>
+          token: ${{ secrets.FLUTTERMIRRORINGBOT_TOKEN }}
```
Comment (godofredoc) [inline .github/workflows/minimal.yml:40]: Please replace with `secrets.FLUTTERACTIONSBOT_CP_TOKEN`

File: .github/workflows/easy-cp.yml (hunk 1 above)
Comment (XilaiZhang) [reply .github/workflows/easy-cp.yml:63]: Sure, thanks for adding the token!

File: .github/workflows/minimal.yml (hunk 2 above)
Comment (XilaiZhang) [reply .github/workflows/minimal.yml]: Sure, thanks for adding the token!

**Review by godofredoc — APPROVED** [review]

**Pull request merged**